File: falleaves-server/auth/src/main/java/cn/falleaves/auth/service/IAuthService.java
package cn.falleaves.auth.service;

import cn.falleaves.auth.model.dto.UserLoginDTO;
import cn.falleaves.auth.model.dto.UserRegisterDTO;
import org.springframework.stereotype.Service;

public interface IAuthService {

    boolean login(UserLoginDTO userLoginDTO) throws Exception;

    boolean register(UserRegisterDTO userRegisterDTO) throws Exception;

    boolean logout() throws Exception;

}

File: falleaves-server/auth/src/main/java/cn/falleaves/auth/service/IAuthService.kt
package cn.falleaves.auth.service

import cn.falleaves.auth.model.dto.UserLoginDTO
import cn.falleaves.auth.model.dto.UserRegisterDTO

interface IAuthService {

    fun login(userLoginDTO: UserLoginDTO): Boolean

    fun register(userRegisterDTO: UserRegisterDTO): Boolean

    fun logout(): Boolean
}